framework/core/tests/integration/api/groups/ListTest.php

```diff
@@ -195,9 +195,9 @@ public function sorts_groups_by_name()
             $this->request('GET', '/api/groups', [
                 'authenticatedAs' => 1,
             ])
-            ->withQueryParams([
-                'sort' => 'nameSingular',
-            ])
+                ->withQueryParams([
+                    'sort' => 'nameSingular',
+                ])
```

Comment:
nitpick: just to keep the git history clean
```suggestion
            ->withQueryParams([
                'sort' => 'nameSingular',
            ])
```